doc/antora/modules/developers/pages/contributing.adoc

```diff
@@ -4,7 +4,7 @@
 
 For a person or company wishing to submit a change to the FreeRADIUS
 project the process can sometimes be daunting if you’re not familiar
-with ``the system.'' This text is a collection of suggestions which can
+with `the system`. This text is a collection of suggestions which can
```

Comment:
I don't see why this should be fixed-width.  It's not referring to a command, debug output, unlang, or anything else like that.

Just use double quotes.